### ISSUE TITLE
TypeError when inline images fail to load

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#1303]
```
Error message: 
can't convert undefined to object

Stacktrace: 
value/this.onbeforeremove/<@https://mail.tutanota.com/main.js:1:166610
a@https://mail.tutanota.com/libs.js:26:73165
[22][22][22]h@https://mail.tutanota.com/libs.js:26:1699
p@https://mail.tutanota.com/libs.js:26:1585
[2]a/this.drainQueues@https://mail.tutanota.com/libs.js:26:1327
```

Tutanota version: 3.52.1

**Comment by armhub** [comment]
Fixed in 535e305e

**Comment by armhub** [comment]
attachments are not visible any more
